rasa/engine/validation.py

```diff
@@ -242,6 +242,7 @@ def _get_parameter_information(
 
     type_hints = _get_type_hints(uses, fn)
     return_type = type_hints.pop("return", inspect.Parameter.empty)
+    type_hints.pop("cls", None)
```

Comment:
Needed to do this because `DIETClassifier.load` now had a type hint (type var) for `cls`, which is not in the function signature. This needs a test I think (I want to validate the approach before writing it)

Reply:
a test would be great

Reply:
1c6ebd9cb75 ✅